tests/__snapshots__/i18n.test.js.snap

```diff
@@ -0,0 +1,63 @@
+/_next/data/%BUILD_ID%/getStaticProps/withRevalidate/withFallback/:id.json  /.netlify/functions/next_getStaticProps_withRevalidate_withFallback_id  200
+/api/shows/:id  /.netlify/functions/next_api_shows_id  200
+/api/shows/:params/*  /.netlify/functions/next_api_shows_params  200
+/api/static  /.netlify/functions/next_api_static  200
+/en/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/en/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/en/getServerSideProps/static  /.netlify/functions/next_getServerSideProps_static  200
+/en/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/en/getStaticProps/static  /.netlify/functions/next_getStaticProps_static  200!  Cookie=__prerender_bypass,__next_preview_data
+/en/getStaticProps/with-revalidate  /.netlify/functions/next_getStaticProps_withrevalidate  200
+/en/getStaticProps/withFallback/:id  /.netlify/functions/next_getStaticProps_withFallback_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/en/getStaticProps/withFallback/:slug/*  /.netlify/functions/next_getStaticProps_withFallback_slug  200!  Cookie=__prerender_bypass,__next_preview_data
+/en/getStaticProps/withRevalidate/:id  /.netlify/functions/next_getStaticProps_withRevalidate_id  200
+/en/getStaticProps/:id  /.netlify/functions/next_getStaticProps_id  200!  Cookie=__prerender_bypass,__next_preview_data
+/en/static/:id  /en/static/[id].html  200
+/es/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/static  /.netlify/functions/next_getServerSideProps_static  200
+/es/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/es/getStaticProps/static  /.netlify/functions/next_getStaticProps_static  200!  Cookie=__prerender_bypass,__next_preview_data
+/es/getStaticProps/with-revalidate  /.netlify/functions/next_getStaticProps_withrevalidate  200
+/es/static/:id  /es/static/[id].html  200
+/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/getServerSideProps/static  /.netlify/functions/next_getServerSideProps_static  200
+/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/getStaticProps/static  /en/getStaticProps/static  200
+/getStaticProps/with-revalidate  /.netlify/functions/next_getStaticProps_withrevalidate  200
+/getStaticProps/withFallback/:id  /en/getStaticProps/withFallback/:id  200
+/getStaticProps/withFallback/:id  /en/getStaticProps/withFallback/:id  200
+/getStaticProps/withFallback/:slug/*  /en/getStaticProps/withFallback/:...slug  200
+/getStaticProps/withFallback/:slug/*  /en/getStaticProps/withFallback/:...slug  200
```

Comment:
double fallback redirect as pointed out in PR body description (also `:...slug` is definitely not correct lol)